src/main/resources/assets/plane/lang/de_de.json

```diff
@@ -0,0 +1,63 @@
+{
+  "entity.plane.plane": "Plane",
```

Comment:
Not translated

Reply:
Wasnt sure if this was the mod name. (misstake i did last time)